docs/semgrep-supply-chain/license-scanning.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: license-scanning
+append_help_link: true
+title: License scanning 
+hide_title: true
+description: "Semgrep Supply Chain can detect and list a package's license. Prevent or exempt certain packages from being used based on its license."
+tags:
+  - Team & Enterprise tier
+  - Semgrep Supply Chain
+---
+
+# License scanning
+
+Semgrep Supply Chain's **license scanning** feature enables you to explicitly allow or disallow (block) a package's use in your repository based on its license. For example, your company policy may disallow the use of packages with the Creative Commons Attribution-NonCommercial (CC-BY-NC) license.
+
+![Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed](/img/sc-license-scanning.png)
+*Figure 1*. Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed.
+
+:::info Prerequisites
+* License scanning can accessed through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to view and manage license workflows.
+* To detect licenses, you must [add or onboard a project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+:::
+
+## Viewing licenses
+
+To view a package's license:
+
+1. [Sign in to Semgrep Cloud Platform](https://semgrep.dev/login).
+2. Click **[Supply Chain](https://semgrep.dev/orgs/-/supply-chain)** > **Dependencies**. By default licenses are listed in the row of their respective package.
+3. If you don't see any licenses:
+    1. Click the Supply Chain **Settings** on the header menu. These settings are specific to Semgrep Supply Chain.
```

Comment:
For TWs: 
I use the term "header menu" to avoid the term "top menu" (position based, disallowed by Google Style Guide). 

"Header" is also a semantic html tag (<header>) so while it still implies a position it is less overt than "top". Finally it is accompanied by a screenshot.